src/components/apps/launch/ResourceRequirements.js

```diff
@@ -168,87 +178,90 @@ const ResourceRequirementsForm = withI18N(
         const classes = useStyles();
 
         return (
-            <ExpansionPanel defaultExpanded>
-                <ExpansionPanelSummary
-                    expandIcon={
-                        <ExpandMore
-                            id={buildDebugId(baseId, ids.BUTTONS.EXPAND)}
-                        />
-                    }
-                >
-                    <Typography variant="subtitle1">
-                        <ResourceRequirementsHeader headerMessageKey="resourceRequirements" />
-                    </Typography>
-                </ExpansionPanelSummary>
+            <>
+                <Typography variant="body1">
```

Comment:
This `Typography` used to be in the `ExpansionPanelDetails` - also not sure if this move was intentional or not

Reply:
- Multi-step with my grid changes:
![image](https://user-images.githubusercontent.com/1250790/86951310-55737b80-c11f-11ea-8d34-6aaadc0c2c91.png)

Reply:
Ok I was able to get it where it was originally by adding `component="span"` to that `Typography`. I will push that change. Here is a screenshot of this change:

![image](https://user-images.githubusercontent.com/1250790/86953111-f2cfaf00-c121-11ea-9d9b-dad87a4b2e47.png)

Reply:
Also `direction="column"` didn't work

Reply:
Ah yeah, since only the form fields are in a grid now, but the help message is still outside that grid.
Oh well, no problem then.